### DISCUSSION TITLE
Segmentation fault is thrown when trying to run app on shared hosting

### DISCUSSION THREAD

**Discussion by fullstackjunior** [pocketbase/pocketbase#4039] [Q&A] [answered]
Hello everyone!

Pocketbase seems to be really nice and fun to develop with and at the moment I'm trying to deploy it on shared hosting with ability to connect to terminal via ssh. But unfortunately facing an issue. When I type the command `pb/pocketbase serve` I get the error `Segmentation fault`

The os version is Debian GNU/Linux 10 (buster)
The Pocketbase version is 0.20.1_linux_amd64

Is there some way to handle this issue? Any help or advice would be very appreciated

**Comment by ganigeorgiev** [comment, marked as answer]
What shared hosting do you use?
I'm not sure if your error is related to that, but most shared hosting providers I've used in the past (including those that allows connecting with ssh) **don't allow running user binaries or opening ports** for security reasons. You usually will need a VPS provider (eg. Hetzner Cloud, Digital Ocean, Vultr, etc.) or hosts with docker/containers support (eg. Fly).

**Reply by fullstackjunior** [reply]
Thank you very much for fast reply!
It's pretty ordinary regional web hosting for local clients.
I have permissions to read, write and execute files, but I didn't think about ports. I will ask hosting support about it.

**Reply by fullstackjunior** [reply]
Okay, they said it's not possible to use 80 or 443 port, so I guess that's the reason. Then I'll try to use VPS.
Thanks again for your help and thanks for developing such cool things like Pocketbase!
